context/app/static/js/components/entity-search/Search/Search.jsx

```diff
@@ -32,7 +32,7 @@ function Search() {
       query: new MultiMatchQuery({
         fields: ['all_text'],
       }),
-      facets: facets.map((facet) => createFacet(facet)),
+      facets: Object.values(facets).map((facet) => createSearchkitFacet(facet)),
```

Comment:
Not new with this PR, but it could be confusing for `facets` to be an Object in some contexts, and an Array in others. Not sure which name should change and which should stay.

Reply:
The name `facets` here is required by searchkit so the name for facets in our provider will have to change.